runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegisterView/UsersRegisterView.styles.ts

```diff
@@ -40,7 +40,7 @@ export const DeleteButton = styled(Button)(({theme}) => `
         && {
             margin-left: 20px;
         }
-        
+
```

Comment:
the best extensions - changes nothing to more nothing
![RaingumpKtoGIF](https://github.com/runbotics/runbotics/assets/69820069/196e65b1-84c3-4b7c-8fa6-e5f55dddf5c5)